### PR TITLE
Update old links to point to the newer ones

### PR DESCRIPTION
**Pull request by ivanm-gdp**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ This repository focuses on examples in Python and only covers common use cases,
 
 ### Get Projects
 
-- The detailed explanation can be accessed [here](https://datasaurai.gitbook.io/datasaur/api/get-data/get-list-of-projects).
+- The detailed explanation can be accessed [here](https://docs.datasaur.ai/api/get-data/get-list-of-projects).
```
Comment (muhammadzpw) [inline readme.md:42]: link di gitbooknya salah seharusnya tanpa "s" https://api-docs.datasaur.ai/#query-getProject
image
oiya minor ttg gitbooknya lagi "_You can easily...._" maunya italic tapi ga bener ya

Comment (ivanm-gdp) [reply readme.md:42]: ah, link di Gitbook ya, oke
updatenya bukan di repo ini, jadi I'll note to others ya 👍🏼

**Review by muhammadzpw — APPROVED** [review]
Updated links are legit, except the one in GitBook :+1:

File: readme.md (hunk 1 above)
Comment (ivanm-gdp) [reply readme.md:42]: eh, tapi query getProjects ada juga ka Rudi 🤔 
https://api-docs.datasaur.ai/#query-getProjects

getProject untuk retrieve 1

getProjects retrieve list-of-project, based on filter. 

---

anyway, buat italic-nya tetep ku raise ya

Comment (muhammadzpw) [reply readme.md:42]: Oh iya my bad.. tadi aku coba soalnya ga langsung ke section getProjects nya kykny masih loading

**Pull request merged**